autoPyTorch/pipeline/tabular_classification.py

```diff
@@ -91,12 +91,7 @@ def _predict_proba(self, X: np.ndarray) -> np.ndarray:
         loader = self.named_steps['data_loader'].get_loader(X=X)
         pred = self.named_steps['network'].predict(loader)
         if isinstance(self.dataset_properties['output_shape'], int):
-            proba = pred[:, :self.dataset_properties['output_shape']]
-            normalizer = proba.sum(axis=1)[:, np.newaxis]
-            normalizer[normalizer == 0.0] = 1.0
-            proba /= normalizer
-
-            return proba
+          return pred
```

Comment:
```suggestion
            # The final layer is always softmax now (it already gives pseudo proba)
            return pred
```

Reply:
On another topic, what do you think the else is doing here in this code block @ravinkohli @nabenabe0928, what would be the usage?

Reply:
its for multilabel multi-class classification. But I don't think we have tested it yet.